test/setProps.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Promisified enzyme function setProps()
+ * Sets the props of given react component
+ * @param component - enzyme
```

Comment:
Are you using this util function in several files?

Reply:
yes, TransformationTest, ContextTest